npm/ng-packs/packages/core/src/lib/services/file-utils.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, inject } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class FileUtilsService {
+  protected readonly document = inject(DOCUMENT);
```

Comment:
[AbpWindowService](https://github.com/abpframework/abp/blob/dev/npm/ng-packs/packages/core/src/lib/services/window.service.ts) we might use this ?